tests/licensedcode/data/plugin_license/text/scan.expected.json

```diff
@@ -107,39 +216,16 @@
               "matcher": "3-seq",
               "license_expression": "fsf-ap",
               "rule_identifier": "fsf-ap.LICENSE",
-              "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/fsf-ap.LICENSE",
-              "referenced_filenames": [],
-              "is_license_text": true,
-              "is_license_notice": false,
-              "is_license_reference": false,
-              "is_license_tag": false,
-              "is_license_intro": false,
-              "rule_length": 35,
-              "rule_relevance": 100,
-              "matched_text": "Reproduction and distribution of this file, with or without modification, are\npermitted in any medium without royalties provided the copyright notice\nand this notice are preserved. This file is offered as-is, without any warranties.",
-              "licenses": [
-                {
-                  "key": "fsf-ap",
-                  "name": "FSF All Permissive License",
-                  "short_name": "FSF All Permissive License",
-                  "category": "Permissive",
-                  "is_exception": false,
-                  "is_unknown": false,
-                  "owner": "Free Software Foundation (FSF)",
-                  "homepage_url": "http://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html",
-                  "text_url": "",
-                  "reference_url": "https://scancode-licensedb.aboutcode.org/fsf-ap",
-                  "scancode_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/fsf-ap.LICENSE",
-                  "spdx_license_key": "FSFAP",
-                  "spdx_url": "https://spdx.org/licenses/FSFAP"
-                }
-              ]
+              "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/licenses/fsf-ap.LICENSE"
```

Comment:
To have reasonably stable URLs will need to update the models to ensure that we never reuse existing rule ids and never rename rules either.... we need to move to a soft-delete mode like with licenses... or could we publish the rules on the license DB too?

Reply:
Yes! we should ensure we don't change rule names (will open an issue for that?), we also have this issue for rules in licenseDB: https://github.com/nexB/scancode-licensedb/issues/3